### ISSUE TITLE
Default Browser?

### ISSUE THREAD

**Issue by diggetybo** [skandasoft/browser-plus#78]
I wouldn't call this an issue per say, but I would really appreciate a dialogue with the developer or the more experienced community that uses browser plus on the following question. Is it possible to set atom's browser plus as the default browser in windows?

I prefer browser plus over chrome, firefox, ie, everything. I'd like to set it as my default browser, but seeing as browser plus is a package within a text editor, windows may not have a GUI solution for such a setting.

Is there some reg edit way I can set browser plus as the default browser? Or maybe I can set Atom as the default "browser" then have browser plus automatically activate upon atom initialization? 

Please advise,

Thank you

**Comment by skandasoft** [comment]
You are right . This could be made into electron based browser ie. a standalone browser. we would need a wrapper around. Anybody willing to do I can help.Right now I don't have the bandwidth.
